eox_core/api/v1/serializers.py

```diff
@@ -11,9 +11,14 @@
 from eox_core.edxapp_wrapper.users import (
     check_edxapp_account_conflicts,
     get_user_read_only_serializer,
+    get_user_signup_source,
     get_username_max_length,
 )
 
+UserSignupSource = get_user_signup_source()  # pylint: disable=invalid-name
+
+MAX_SIGNUP_SOURCES_TO_UPDATE = 1
```

Comment:
This constant is a bit confusing. I thought that you where going to update the signups sources.
What about calling it: MAX_SIGNUP_SOURCES_ALLOWED?

Reply:
agree